### ISSUE TITLE
Stripe virtual cards

### ISSUE THREAD

**Issue by themightychris** [opencollective/opencollective-api#1385]
Stripe is finally getting into the virtual API-driven cards game: https://stripe.com/issuing

Since OpenCollective already uses stripe, does this make the prospect more feasible now of enabling collectives to issue cards to organizers to charge expenses directly to their balance?

**Comment by HipsterBrown** [comment]
Thanks for the notice @themightychris. We have requested an invitation to try out the API with hopes of allowing collectives to issue cards to admins and organizers. It could be a pretty awesome feature for the platform.

**Comment by adreyfus-stripe** [comment]
I work at Stripe, let me know if you want any help with this enhancement!

**Comment by piamancini** [comment]
hey @adreyfus-stripe we've been thinking about this, specially for collectives that are self-hosted and to help collectives that can afford to front the expenses and then get reimbursed. As @HipsterBrown mentions above we have requested an invite to try the API. Not sure if that's accepted already? 
This is not a hight priority at the moment but we can start exploring it.

**Comment by adreyfus-stripe** [comment]
Yep makes sense! Just wanted to make myself available as a resource when the time comes :)

**Comment by piamancini** [comment]
hi @adreyfus-stripe any chance you can help us get on the beta?

**Comment by adreyfus-stripe** [comment]
hi @piamancini! ack-ing that I saw this but I'm catching up on a lot of things after vacation so it might take me a few days to investigate for you 👍

**Comment by znarf** [comment]
Would be great to move this issue to https://github.com/opencollective/opencollective/issues as it's not API/backend specific.
